git/git.go

```diff
@@ -1279,9 +1279,15 @@ func Fetch(remotes ...string) error {
 
 // RemoteRefs returns a list of branches & tags for a remote by actually
 // accessing the remote via git ls-remote.
-func RemoteRefs(remoteName string) ([]*Ref, error) {
+func RemoteRefs(remoteName string, ignoreTags bool) ([]*Ref, error) {
 	var ret []*Ref
-	cmd, err := gitNoLFS("ls-remote", "--heads", "--tags", "-q", remoteName)
+	args := []string{"ls-remote", "--heads", "-q"}
+	if !ignoreTags {
+		args = append(args, "--tags")
+	}
+	args = append(args, remoteName)
+	
```

Comment:
There's some extra trailing whitespace on this line which `go fmt` is detecting -- would you mind removing that?  That should allow our CI suite to pass.